engine/core.py

```diff
@@ -97,7 +97,7 @@ def start_game(self) -> None:
                 (self.width, self.height),
                 self.ingame_gui,
             )
-            self.ingame_state.setup()
+#            self.ingame_state.setup()
```

Comment:
Don't commit commented out code. Just remove the code entirely if it doesn't need to be there.